README.md

```diff
@@ -154,12 +154,6 @@ Pylance provides users with the ability to customize their Python language suppo
         - `true`
         - `false` (default)
 
-- `python.analysis.autoImportUserSymbols`
```

Comment:
This may not be in the package.json but we still read this value in the code. @Heejaechang, I believe you added this for an experiment in this commit: https://github.com/microsoft/pyrx/commit/893f8cf8dd9763f9125a2ecd22b29f88b65a97a7

The `includeUserSymbolsInAutoImport` is still used internally? Should we remove it?

Reply:
it has been there for long time and no one complained about it, so I think we can remove it.